CHANGELOG/CHANGELOG-1.3.md

```
@@ -0,0 +1,964 @@
+#### Docker runtime Known Issues
+
+- Kernel crash with Aufs storage driver on Debian Jessie ([#27885](https://github.com/kubernetes/kubernetes/issues/27885))
+  - Consider running the *new* [kubernetes node problem detector](https://github.com/kubernetes/node-problem-detector) to identify this (and other) kernel issues automatically.
+
+- File descriptors are leaked in docker v1.11 ([#275](https://github.com/docker/containerd/issues/275))
+- Additional memory overhead per container in docker v1.11 ([#21737](https://github.com/docker/docker/issues/21737))
+- [List of upstream fixes](https://github.com/docker/docker/compare/v1.10.3...runcom:docker-1.10.3-stable) for docker v1.10 identified by RedHat
+
+#### Rkt runtime Known Issues
+
+- A detailed list of known issues can be found [here](https://github.com/kubernetes/kubernetes.github.io/blob/release-1.3/docs/getting-started-guides/rkt/notes.md)
```

markdownlint-cli2 (0.22.1)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)